0000-template.md

```diff
@@ -1,6 +1,6 @@
 # Meta
 [meta]: #meta
-- Name: (fill in the feature name: My Feature)
+- Name: TEST
```

Comment:
/queue-issue samj1912/docs "This is a test"

Reply:
/unqueue-issue c33a34

Reply:
Wow, you are really testing it, huh. Not sure how it should handle edits...

Reply:
It doesn't. Other curious thing I noticed - the uid is a hash of the repo and the title and it can sometimes put duplicates there but it manages to delete all of them with unqueue so it works fine 👍 great job! Thanks for creating this! :D 🙇

Reply:
/queue-issue samj1912/docs "This is an edited test"
/queue-issue samj1912/docs "This is an edited test2"

Reply:
/unqueue-issue 5e43de
/unqueue-issue aa4a71